src/open_inwoner/components/templates/components/Filter/Filter.html

```diff
@@ -1,15 +1,18 @@
-{% load i18n filter_tags icon_tags form_tags %}
+{% load i18n filter_tags icon_tags button_tags form_tags %}
 
+{% spaceless %}
```

Comment:
See comment in the Conversation.

Putting spaceless here would just turn this whole structure into a oneliner, why would that be good?